rclpy/executors/test/test_copyright.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ament_copyright.main import main
+
+
+def test_copyright():
+    rc = main(argv=['.'])
```

Comment:
Maybe you've based this off of an existing example, but is this the right argument to pass here? `.` going to be the folder of this script, I guess not, but is it then the directory from which you run the tests? Wouldn't be more correct to give it a relative path from the location of this script so you know it's getting the source code no matter where you run it?

Would apply to the other tests.

Reply:
tbh I copy pasted these from another package without looking at the files. It seems like the arguments here work using `ament test`; however, I agree it's not obvious what's really being passed in. In 3d69927 I implemented your suggestion by making the argument a directory based on the location of the test script. This allows it to be run anywhere and makes it clear what the directory is.

Reply:
The file doesn't have a `main` so can't be invoked directly. It is always invoked using `nosetests3` so the current code is fine as-is.